### PR TITLE
Refactor freshtab

### PR DESCRIPTION
**Pull request by jayeshsolanki93**

### CODE REVIEW

File: app/src/lumen/java/com/cliqz/browser/starttab/freshtab/FreshTab.kt [hunk 1]
```diff
@@ -69,14 +105,25 @@ internal class FreshTab : StartTabFragment(), FreshTabContract.View {
...
+    private fun hideAllTrialPeriodViews() {
         trial_period_lumen_upgrade.visibility = View.GONE
         trial_over_lumen_upgrade.visibility = View.GONE
     }
 
-    override fun toggleWelcomeMessage(show: Boolean) {
-        welcome_image.visibility = if (show) View.VISIBLE else View.GONE
-        welcome_title.visibility = if (show) View.VISIBLE else View.GONE
+    private fun toggleWelcomeMessage() {
+        welcome_image.visibility = if (isFreshInstall) View.VISIBLE else View.GONE
```
Comment (ravjit-cliqz) [inline app/src/lumen/java/com/cliqz/browser/starttab/freshtab/FreshTab.kt]: Lets try to follow the [kotlin style guidelines](https://developer.android.com/kotlin/style-guide) by android
Use braces around single line if else

File: app/src/lumen/java/com/cliqz/browser/starttab/freshtab/FreshTab.kt [hunk 2]
```diff
@@ -69,14 +105,25 @@ internal class FreshTab : StartTabFragment(), FreshTabContract.View {
...
+        welcome_title.visibility = if (isFreshInstall) View.VISIBLE else View.GONE
```
Comment (ravjit-cliqz) [inline app/src/lumen/java/com/cliqz/browser/starttab/freshtab/FreshTab.kt]: duplicate line

File: app/src/lumen/java/com/cliqz/browser/starttab/freshtab/FreshTab.kt [hunk 3]
```diff
@@ -26,25 +31,56 @@ internal class FreshTab : StartTabFragment(), FreshTabContract.View {
...
+        toggleWelcomeMessage()
+        if (purchasesManager.purchase.isASubscriber) {
+            hideAllTrialPeriodViews()
+        } else {
+            getTrialPeriod()
+        }
+    }
+
+    private fun getTrialPeriod() {
+        // Hacky solution with handler. Bus wasn't working, giving a DispatcherNotFoundException
```
Comment (ravjit-cliqz) [inline app/src/lumen/java/com/cliqz/browser/starttab/freshtab/FreshTab.kt]: Bus should work, lets look into it

File: app/src/lumen/java/com/cliqz/browser/starttab/freshtab/FreshTab.kt (hunk 1 above)
Comment (jayeshsolanki93) [reply app/src/lumen/java/com/cliqz/browser/starttab/freshtab/FreshTab.kt]: It's an expression and fits in one line:
https://developer.android.com/kotlin/style-guide#expressions

**Comment by ravjit-cliqz** [comment]
move this to master

**Pull request merged**